lib/bot.js

```diff
@@ -310,6 +322,14 @@ class Bot {
     }
   }
 
+  getDiscordAvatar(nick) {
+    const user = this.discord.users.find('username', nick);
+    if (user) {
+      return user.avatarURL;
+    }
+    return null;
+  }
+
```

Comment:
I don't think case insensitivity should be a blocker but it does make me wonder what happens if two Discord users have the same display name, as this seems doable, or the same name ignoring case. This is a problem elsewhere already, I guess, such as when an IRC user mentions a Discord user.

This function also currently checks by username as opposed to nickname (display name), which might be unexpected. This should maybe:

1. Check by nickname and username
2. Check by the above, case-insensitively, if no match found
3. If more than one found, default to no fancy icon

That might be more complicated than necessary, but I think it would suffice for most or all use-cases. It should at least check nicknames?